### ISSUE TITLE
Restore to new user?

### ISSUE THREAD

**Issue by colintedford** [bit-team/backintime#499]
I'm finally about to update my OS, but I'd planned to use a different (shorter) username & hostname -- what will I need to do to restore my files and BiT profiles? I know I can manually copy (or rsync) them from the backup folders, but would like to restore through BiT if possible. I'm backing up my dotfiles, so I will have a copy of ~/.config/backintime/config.

I'm using Back In Time 1.0.7 on LXLE 12.04 (based on Lubuntu 12.04), and moving to whatever version of Back In Time is on LXLE / Lubuntu 14.04. The update isn't an in-place upgrade; I have to do a fresh installation. Also, some of my profiles backup (and all of them backup to) external hard drives.

(I don't see how to add a label, but obviously this is a question).

**Comment by Germar** [comment]
As long as you use BIT version > 1.1.0 it will restore the config automatically on first start on the new machine. Also different host/usernames doesn't matter anymore (the old `Auto Host/User/ProfileID` option is gone). 

You should `fsck` your backup drive and make sure your backup is healthy by making one last snapshot with `backintime --checksum --backup`. This will take ages but after that you know all files are really the same as original files (could happen that a filesystem error broke a file which wasn't detected by rsyncs normal method)

I'd suggest to also backup `/etc` as root if you configured something special in there. And create a backup of your installed packages. Take a look on our FAQ [How to backup Debian/Ubuntu Package selection](https://github.com/bit-team/backintime/wiki/FAQ#how-to-backup-debian-ubuntu-package-selection) and the according [restore process](https://github.com/bit-team/backintime/wiki/FAQ#how-to-restore-debian-ubuntu-package-selection)

**Comment by colintedford** [comment]
Thank you! Does `backintime --backup` run all profiles that are setup? I checked the man page but didn't see any mention. I know I can temporarily change my profiles in the GUI to use checksum, anyway. 

I've never run `fsck` manually before, and my first look at the man page and a web search made me more nervous instead of less... I will do the other steps, and make another copy of my files, and hopefully I won't take the impatient route of skipping the `fsck`.

**Comment by Germar** [comment]
No, `backintime --backup` will only run the first profile. Use `backintime --profile-id 2 --backup` for the second.

Don't fear the shell ;-) Just use `gparded`. Select your partition, go to `Partition > Check` and run the check with `Edit > Apply All Operations`.

**Comment by colintedford** [comment]
Ah, <code>gparted</code> I've used before (though not for that). Thanks again!

I'm a lot more comfortable with the shell than I used to be -- but there's a lot I don't know and sometimes it's easier (and safer) to back away than to climb the learning curve :)

**Comment by colintedford** [comment]
I ended up using the GUI to check the backups because one of the first outputs I got in the shell was <code>Ignore option: --checksum</code>. After a confused upgrade, I ended up not needing to restore the files, just the profiles. I had to use the PPA because the version in the repo was older than 1.1.0. I also had to update the snapshot locations for every profile because my system now mounts external drives under <code>/media/[username]</code> instead of just <code>/media</code>.

The only question I have now is: do I need to change the info under <code>Settings > General > Advanced</code> for the profiles? Currently they show my old host and user. The ext4 drives are owned by my current user now, if that matters.
